dmoj/cptbox/helper.h

```diff
@@ -6,6 +6,7 @@
 #define PTBOX_SPAWN_FAIL_SECCOMP        203
 #define PTBOX_SPAWN_FAIL_TRACEME        204
 #define PTBOX_SPAWN_FAIL_EXECVE         205
+#define PTBOX_SPAWN_FAIL_LANDLOCK       206
```

Comment:
Needs some new handling code in https://github.com/DMOJ/judge-server/blob/eefbcbba4194fea472ef51287396b38682af24cc/dmoj/cptbox/tracer.py#L207-L221 too